### ISSUE TITLE
[bug] pouch logs panic

### ISSUE THREAD

**Issue by Letty5411** [alibaba/pouch#1105]
### Ⅰ. Issue Description
pouch client panic.
### Ⅱ. Describe what happened
```
#pouch run -d busybox echo test
de57d25e2e37a97b4300cd5839d27895398d1ac8d7c8de3d2e0b51d807f8e44a

#pouch logs de57d25e2e37a97b4300cd5839d27895398d1ac8d7c8de3d2e0b51d807f8e44a
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xe8683d]

goroutine 1 [running]:
main.(*LogsCommand).runLogs(0xc4204d4540, 0xc4204eb8a0, 0x1, 0x1, 0x0, 0x0)
	/tmp/pouchbuild/src/github.com/alibaba/pouch/cli/logs.go:71 +0xfd
main.(*LogsCommand).Init.func1(0xc4200ee900, 0xc4204eb8a0, 0x1, 0x1, 0x0, 0x0)
	/tmp/pouchbuild/src/github.com/alibaba/pouch/cli/logs.go:40 +0x48
github.com/alibaba/pouch/vendor/github.com/spf13/cobra.(*Command).execute(0xc4200ee900, 0xc4204eb870, 0x1, 0x1, 0xc4200ee900, 0xc4204eb870)
	/tmp/pouchbuild/src/github.com/alibaba/pouch/vendor/github.com/spf13/cobra/command.go:742 +0x49b
github.com/alibaba/pouch/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc4202686c0, 0xc42028ded0, 0x1, 0x1)
	/tmp/pouchbuild/src/github.com/alibaba/pouch/vendor/github.com/spf13/cobra/command.go:827 +0x30e
github.com/alibaba/pouch/vendor/github.com/spf13/cobra.(*Command).Execute(0xc4202686c0, 0x1, 0xc4204eb800)
	/tmp/pouchbuild/src/github.com/alibaba/pouch/vendor/github.com/spf13/cobra/command.go:780 +0x2b
main.(*Cli).Run(0xc42058ec80, 0x16486e0, 0xc4204eadf0)
	/tmp/pouchbuild/src/github.com/alibaba/pouch/cli/cli.go:78 +0x2f
main.main()
	/tmp/pouchbuild/src/github.com/alibaba/pouch/cli/main.go:49 +0x96c
```

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1.
2.
3.

### Ⅵ. Environment:

- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by Letty5411** [comment]
It seems that this panic id due to LogConfig is nil?
```
if !validDrivers[c.HostConfig.LogConfig.Type] {
		return fmt.Errorf("\"logs\" command is supported only for \"json-file\" and \"journald\" logging drivers (got; %s)", c.HostConfig.LogConfig.Type)
	}

```

**Comment by oiooj** [comment]
@Letty5411 Could you help me close this issue manually? I forgot to reference this before #1192 merged. Sorry about that.

**Comment by HusterWan** [comment]
resolved by #1192
